axual-webclient/applications_access_grant_data.go

```diff
@@ -252,52 +252,29 @@ type GetApplicationAccessGrantsByAttributeResponse struct {
 					Type             string      `json:"type"`
```

Comment:
Why did the response structure in GetApplicationAccessGrantsByAttributeResponse change? I understand why you had to change it but why PM now returns a different structure than before?

Reply:
Checked together with @Farnaz-java-dev , The API didn't change. The original struct wasn't correct and I have pushed the changes